policy/policy.go

```diff
@@ -267,9 +267,26 @@ func (r *RepoPolicy) CreatePR(bundle, title, baseBranch string, dryRun bool) (st
 		}
 		log.Info().Str("baseBranch", baseBranch).
 			Str("title", title).Msg("calling CreatePR on github")
-		prURL, err = r.gitRepo.CreatePR(baseBranch, title, string(body))
+		pr, err := r.gitRepo.CreatePR(baseBranch, title, string(body))
 		if err != nil {
 			return "", err
+		} else {
```

Comment:
Is this else required? The return in the other branch ensures that control will never reach this point in the case of an error.